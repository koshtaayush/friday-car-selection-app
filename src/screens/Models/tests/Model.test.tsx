import React from 'react'
import { shallow } from 'enzyme'
import Model from '..'

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useHistory: () => ({
        push: jest.fn(),
    }),
}))

describe('Model', () => {

    it('Should match the snapshot', () => {
        const component = shallow(
            <Model />
        )

        expect(component).toMatchSnapshot()
    })
})
