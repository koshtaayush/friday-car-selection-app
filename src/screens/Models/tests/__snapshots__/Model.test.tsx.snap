// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model Should match the snapshot 1`] = `
<Fragment>
  <Search
    onChangeProp={[Function]}
    placeholder="Search for a model"
    value=""
  />
  <styled.div />
  <ScreenMessenger
    primaryMessage="No models found"
    secondaryMessage="Please try to search for another make"
  />
</Fragment>
`;
